configs/pipeline/2021.05.01.open.ini

```diff
@@ -0,0 +1,60 @@
+[DEFAULT]
+version = 2021.05.01.open
+description = Config for the open data version of the tech grants pipeline. Uses only openly available (360Giving grants data) datasets to create training data, train models and evaluate. RF or EPMC data evaluated on was created by running `python nutrition_labels/create_training_data.py --config_path configs/training_data/2021.03.29.epmc.ini` and `python nutrition_labels/create_training_data.py --config_path configs/training_data/2021.03.29.rf.ini` - these create data/processed/training_data/210329epmc/training_data.csv and data/processed/training_data/210329rf/training_data.csv respectively.
+
+[data]
+# Variables used in create_training_data.py
+epmc_tags_query_one_filedir = data/raw/expanded_tags/EPMC_relevant_tool_pubs_3082020.csv
+epmc_tags_query_two_filedir = data/raw/expanded_tags/EPMC_relevant_pubs_query2_3082020.csv
+epmc_pmid2grants_dir = data/raw/EPMC/pmid2grants.json
+rf_tags_filedir = data/raw/expanded_tags/research_fish_manual_edit.csv
+grant_tags_filedir = data/raw/expanded_tags/wellcome-grants-awarded-2005-2019_manual_edit_Lizadditions.csv
+prodigy_filedir = data/prodigy/merged_tech_grants/merged_tech_grants.jsonl
+grant_data_filedir = data/raw/wellcome-grants-awarded-2005-2019.csv
+sources_include = ['Prodigy grants', 'Grants']
+# Variables used in grant_tagger.py to train model
+training_data_file = data/processed/training_data/210501open/training_data.csv
+label_name = Relevance code
+training_data_file_id = Internal ID
+prediction_cols = ['Title', 'Description']
```

Comment:
yeh not the most elegant! This is because these new config files are a combination of 4 different config files - so they were all designed with different scripts in mind and _then_ combined as one. There could certainly be more unity in these types of things if redesigned. In this case I wrote `create_training_data.py` in one way, and then `predict.py` and `evaluate.py` in another way.